commit e56b2ead4021e94eb15f8dd7240c607fe61ce57f
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Fri May 29 11:18:58 2015 +0100

    scripts/knlinfo: Decode DDTK atom
    
    Show the DDTK atom as being a boolean.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/scripts/knlinfo b/scripts/knlinfo
index b9ef124..263ec93 100755
--- a/scripts/knlinfo
+++ b/scripts/knlinfo
@@ -16,6 +16,7 @@ my $trailer_magic = 'RPTL';
 
 my %atom_formats =
 (
+    'DDTK' => \&format_bool,
     'DTOK' => \&format_bool,
     'KVer' => \&format_string,
     '270X' => \&format_bool,
@@ -148,7 +149,7 @@ sub format_atom
 sub format_bool
 {
     my ($data) = @_;
-    return unpack('V', $data) ? 'true' : 'false';
+    return unpack('V', $data) ? 'y' : 'n';
 }
 
 sub format_int
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1181_e56b2ead4021e94eb15f8dd7240c607fe61ce57f.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1181_e56b2ead4021e94eb15f8dd7240c607fe61ce57f.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
